tests/test_taar_lite_guidguid.py

```diff
@@ -23,17 +23,31 @@
     Row(installed_addons=["test-guid-1", "test-guid-1"])
 ]
 
-MOCK_ADDON_INSTALLATIONS = {
-    "test-guid-1":
-        {"test-guid-2": 1,
-         "test-guid-3": 2,
-         "test-guid-4": 2
-         },
-    "test-guid-2":
-        {"test-guid-1": 2,
-         "test-guid-5": 1,
-         "test-guid-6": 1
-         }}
+MOCK_ADDON_INSTALLATIONS = [
```

Comment:
A better name might be `EXPECTED_ADDON_INSTALLATIONS`, since mocking is more generally faking data from sources outside of the script's control. You may also consider making this a dataframe, and using the `df_equals` test-fixture found in `conftest.py`. Example usage in this repo:

https://github.com/mozilla/python_mozetl/blob/054b34e93f687d344c1b3e9ab613681166858ef1/tests/engagement/test_daily_retention.py#L69-L85